applications/osb-portal/src/App.tsx

```diff
@@ -42,8 +42,8 @@ export const App = (props: any) => {
                 <ProtectedRoute exact={true} path="/workspace/:workspaceId">
                   <WorkspacePage />
                 </ProtectedRoute>
-                <ProtectedRoute exact={true} path="/workspace/:workspaceId/:app">
-                  <WorkspacePage />
+                <ProtectedRoute exact={true} path="/workspace/open/:workspaceId/:app">
```

Comment:
the workspace page is a public page. Need to control inside the workspace is the anonymous user has the right to see the workspace (which is true for public/featured workspaces). Can be reused some of the logic in the WorkspaceOpenPage